app/components/web-components/vl-decisions-column.js

```diff
@@ -1,21 +1,49 @@
-// TODO: octane-refactor
-// eslint-disable-next-line ember/no-classic-components
-import Component from '@ember/component';
-import { computed } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
 
-// TODO: octane-refactor
-// eslint-disable-next-line ember/no-classic-classes, ember/require-tagless-components
-export default Component.extend({
+export default class WebComponentsVlDecisionsColumn extends Component {
+  @service store;
 
-  textToShow: computed('row', 'value', 'row.agendaActivity.subcase.decisions.[]', async function() {
-    const agendaitem = await this.row;
-    const agendaActivity = await agendaitem.get('agendaActivity');
-    const subcase = await agendaActivity.get('subcase');
-    const approved = await subcase.get('approved');
+  @tracked textToShow;
+
+  constructor() {
+    super(...arguments);
+
+    this.loadTextToShow.perform();
+  }
+
+  @task
+  *loadTextToShow() {
```

Comment:
Use `SubcaseIsApprovedService` or does this contain different logic? In that case, same comments as in `SubcaseIsApprovedService` apply here.